.github/workflows/nightly-ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        java-version: [ 11, 17, 20 ]
+        java-version: [ 11, 21-ea ]
```

Comment:
is it possible to keep the JDK 17 in matrix ?

Reply:
- I've added back JDK17.